src/agent/application.cpp

```diff
@@ -93,7 +96,9 @@ void Application::Init(const std::string &aRestListenAddress, int aRestListenPor
 {
     mHost.Init();
 
-    switch (mHost.GetCoprocessorType())
+    CoprocessorType type = mHost.GetCoprocessorType();
```

Comment:
Suggest moving this to after `{`. We generally aim to place all local variable definitions at the top of the scope (start of `{}`).